### PR TITLE
Owls 95882 part 1 - Fixes for the stop server script, changes to capture script output and make grace period dynamic

### PR DESCRIPTION
**Pull request by ankedia**
Owls 95882 part 1 - This PR has couple of fixes for the stop server script, (1) there was a right brace missing and (2) shutdown call was being made with unexpected argument properties. In addition, I have made changes to capture the script output in the exited container logs. Finally, it sets the pod deletion grace period seconds to 0 when the server has already shutdown.

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/11634/

### CODE REVIEW

File: operator/src/main/resources/scripts/stopServer.sh [hunk 1]
```diff
@@ -113,3 +113,7 @@ if [ -f ${SERVER_PID_FILE} ]; then
 fi
 
 trace "Exit script"  &>> ${STOP_OUT_FILE}
+
+trace "Stop Server: === Contents of the script output file ${STOP_OUT_FILE} ===" > /proc/1/fd/1
```
Comment (tbarnes-us) [inline operator/src/main/resources/scripts/stopServer.sh:117]: What is the purpose of ` >> /proc/1/fd/1`?

I recommend at least adding a comment.

Comment (ankedia) [reply operator/src/main/resources/scripts/stopServer.sh:117]: K8S does not print any logs in PreStop stage. PreStop is part of lifecycle, also called as hook and generally hook logs will not print in kubectl logs. See issue [25766 ](https://github.com/kubernetes/kubernetes/issues/25766) for more details. The workaround is to print hook's output to the main process' stdout using `/proc/1/fd/1` as mentioned in the [last comment](https://github.com/kubernetes/kubernetes/issues/25766#issuecomment-752121781) in the above issue. I'll add the comment.

Comment (jshum2479) [reply operator/src/main/resources/scripts/stopServer.sh:117]: I see the script is always appending to ${STOP_OUT_FILE},  what if this file is on PV?  Will it be confusing?

Comment (ankedia) [reply operator/src/main/resources/scripts/stopServer.sh:117]: ok. I'm not sure why it's always appending to ${STOP_OUT_FILE}. Do you think we should instead overwrite the ${STOP_OUT_FILE} file for each run?

Comment (tbarnes-us) [reply operator/src/main/resources/scripts/stopServer.sh:117]: as you mentioned in the scrum, rotating it seems like a good idea - you can use the same utils helper function that we use for other log files...

Comment (ankedia) [reply operator/src/main/resources/scripts/stopServer.sh:124]: Changed in [aa5c19d](https://github.com/oracle/weblogic-kubernetes-operator/pull/3283/commits/aa5c19d141a225a9a69d5ac56af1e691c398b548).

**Review by jshum2479 — APPROVED** [review]

**Review by tbarnes-us — APPROVED** [review]

**Review by lennyphan — APPROVED** [review]

**Pull request merged**